roles/kubernetes/preinstall/tasks/verify-settings.yml

```diff
@@ -0,0 +1,71 @@
+---
+- name: Stop if ansible version is too low
+  assert:
+    that:
+      - ansible_version.full|version_compare('2.3.0.0', '>=')
+  run_once: yes
```

Comment:
Can we also check jinja version as well? Maybe >2.9 ?

Reply:
I want to, but it's not exposed through facts and I can't find any examples. Maybe you have some creative ideas?